.env

```diff
@@ -1,7 +0,0 @@
-CLIENT_ID_LINKEDIN: 86t9ro4djktxl5
-CLIENT_SECRET_LINKEDIN: 3KBxNpF1xj7pxNnJ
-
-CLIENT_ID_GITHUB: 17fbb6c1cd49eb981c43
-CLIENT_SECRET_GITHUB: 26dde7e190c02bb5a73c1aad4e3687853787b1ca
```

Comment:
this is still in the git repo

Reply:
https://docs.github.com/en/github/managing-large-files/removing-files-from-a-repositorys-history

Reply:
when I git rm --cached .env, it couldn't find the file

Reply:
cuz you deleted it here. create a new .env file and add it. I think that should fix the issue.